### DISCUSSION TITLE
pnpm

### DISCUSSION THREAD

**Discussion by alexnguyennz** [shishkin/astro-pagefind#53] [Q&A] [answered]
Hi is it possible for this integration to work with pnpm? I get issues building with it but not with npm. Thanks.

**Comment by shishkin** [comment]
Hi @alexnguyennz! I don't use pnpm personally so I'm not aware of the issues it may have. Can you please elaborate what issues you see?

**Comment by alexnguyennz** [comment]
Sure basically the build process gets right to the end after all the files and images, etc. are outputted, with this error: `Command failed: npx pagefind --site "path"` since it's not npm. I'm not sure how other libraries handle this with different package managers.

**Comment by alexnguyennz** [comment, marked as answer]
Have found out this is to due with Pagefind specifically (I tried integrating it manually), so will close this.

**Reply by shishkin** [reply]
Glad you solved it. Would you mind posting here your solution so it can help someone else?

**Reply by alexnguyennz** [reply]
I ended up just switching to npm.
